templates/home.twig

```diff
@@ -135,7 +135,7 @@
 
 <div>
 	<div class="page-header text-center">
-		<h2>Featuring</h2>
+		<h2>Featuring.</h2>
```

Comment:
I don't think a full stop makes sense here, the other changes are on point.

Reply:
Suggestion: what about a ":" instead @Umcaruje ?